odk2-src/services-managing.rst

```diff
@@ -0,0 +1,303 @@
+Managing ODK Services
+==============================
+
+.. _services-managing:
+
+.. contents:: :local:
+
+.. _services-architect-prereqs:
+
+Prerequisites
+---------------------
+
+ODK Services is a prerequisite to all Data Management Application. You will also need the ODK tools:
+
+  - :doc:`app-designer-intro`
+  - :doc:`cloud-endpoints-intro`
+
+You will also need at least one of the following ODK tools:
+
+  - :doc:`survey-intro`
+  - :doc:`tables-intro`
+
+Survey and Tables can work independently and do not require you to use both.
+
+Finally, you will need the third party apps:
+
+- `OI File Manager <https://play.google.com/store/apps/details?id=org.openintents.filemanager>`_
+
+If you have not installed Services already, follow our guide for :doc:`services-install`
+
+.. _services-managing-servers:
+
+Compatible Servers
+~~~~~~~~~~~~~~~~~~~~~~
+
+The latest version of ODK Services is Version 2.0.3 (February 2018).
```

Comment:
Maybe we should point users to the releases page instead, so that we won't need to update this every release?